packages/x-charts/src/models/axis.ts

```diff
@@ -133,19 +134,55 @@ export interface ChartsAxisProps extends TickParams {
 }
 
 export interface ChartsYAxisProps extends ChartsAxisProps {
-  /**
-   * Position of the axis.
-   */
-  position?: 'left' | 'right';
+  axis?: 'y';
```

Comment:
This prop is only to differentiate between X/Y axis types. It has no effect.

This is necessary since `position`, which was the different property, got moved to the axis config instead.